shared/oval/dir_perms_etc_httpd_conf.xml

```diff
@@ -3,12 +3,14 @@
     <metadata>
       <title>Directory /etc/httpd/conf/ Permissions</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
+        <platform>multi_platform_rhel</platform>
       </affected>
       <description>Directory permissions for /etc/httpd/conf/ should be set to 0750 (or stronger).</description>
-      <reference source="JL" ref_id="20140403" ref_url="test_attestation"/>
+      <reference source="galford" ref_id="20160120" ref_url="test_attestation"/>
     </metadata>
-    <criteria>
+    <criteria operator="OR">
+      <extend_definition comment="httpd not present or in use"
+      definition_ref="package_httpd_removed" />
       <criterion test_ref="test_dir_perms_etc_httpd_conf" />
     </criteria>
   </definition>
```

Comment:
Here the mode check should also allow stronger permissions (sorry for not catching that immediately). Just remove all the 'true'  lines in the ```state_dir_perms_etc_httpd_conf``` state below (since they are irrelevant). When they are set the mode is 0750, if they aren't set the mode is stronger. What matters being all those "false" valued lines in the state below.